cpp/arcticdb/python/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})    
+    message(FATAL_ERROR "This file should not be included directly yet. Use the top-level CMakeLists.txt instead.")
+    find_package(arcticdb REQUIRED)
+    if(${ARCTICDB_USING_CONDA})
+    SET(ARCTICDB_CORE_LIBRARY arcticdb::arcticdb_core_shared)
+    else()
+        SET(ARCTICDB_CORE_LIBRARY arcticdb::arcticdb_core_static)
+    endif()
+else()
+    SET(ARCTICDB_CORE_LIBRARY arcticdb::arcticdb_core_static)
+endif()
+
+    
+## Core python bindings, private only ##
+set(arcticdb_python_srcs
+        ../async/python_bindings.cpp
+        ../codec/python_bindings.cpp
+        ../column_store/python_bindings.cpp
+        ../storage/python_bindings.cpp
+        ../stream/python_bindings.cpp
+        ../toolbox/python_bindings.cpp
+        ../version/python_bindings.cpp)
+
+add_library(arcticdb_python STATIC ${arcticdb_python_srcs})
+
+target_link_libraries(arcticdb_python
+        PUBLIC
+        #tsan
+        arcticdb_core_static
+        ${AWSSDK_LINK_LIBRARIES}
```

Comment:
I know this is copy/paste and it's somewhat outside of the scope, but I think that AWSSDK_LINK_LIBRARIES should be PRIAVTE